src/test/travis.pre.sh

```diff
@@ -12,8 +12,16 @@ deb-src http://download.tarantool.org/tarantool/1.9/ubuntu/ $release main
 EOF
 
 sudo apt-get update
-sudo apt-get -y install tarantool tarantool-common
+sudo apt-get -y install tarantool tarantool-common inetutils-inetd
 
 sudo cp src/test/instance.lua /etc/tarantool/instances.enabled/jdk-testing.lua
 sudo tarantoolctl stop  example
 sudo tarantoolctl start jdk-testing
+
+# Create a network endpoint for tarantoolctl in order to give java tests
+# a possibility to start and stop tarantool instances without being root.
+sudo update-inetd --group OTHER --add "tntctl stream tcp nowait root /bin/bash bash -c 'export COLUMNS=256;read;tarantoolctl \$REPLY'"
```

Comment:
Reworked this stuff to follow your proposition.

Reply:
I think we can further simplify steps needed to run tests: let java manage instances using tarantooctl. I proposed changes on `gh-51-add-basic-connector-tests.start-instance-from-test` branch. It can be a bit dirty in context of unused imports and so, because I don't have right tooling for java for now. Feel free to adopt it as you wish.

The proposed change slightly increases time to run tests, because the jdk-testing instance starts and stops for each test file. Don't sure it is right thing to do, maybe this code should be run somehow from pom.xml before and after the whole test suite. From the other side, this approach looks more clear in context of possible future multi-instance tests.

Also it would be better to output tarantool log file right from Java code (in `@AfterAll` method) instead of `cat testroot/jdk-testing.log` in .travis.yml.

What do you think about that at all?

Reply:
Great! So, can you integrate this proposal here? I trust your hands more then mine, at least while I don't have appropriate enviroment for Java development (static checkers, IDE and so on). Sorry that this thing increare your effort.